hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/GetServiceRolesSubcommand.java

```diff
@@ -76,4 +90,24 @@ private void getOmServerRoles(List<ServiceInfo> serviceList) {
       }
     }
   }
+
+  private void getOmServerRolesAsJson(List<ServiceInfo> serviceList)
+      throws IOException {
+    List<Map<String, Map<String, String>>> omServiceList = new ArrayList<>();
+    for (ServiceInfo serviceInfo : serviceList) {
+      OMRoleInfo omRoleInfo = serviceInfo.getOmRoleInfo();
+      if (omRoleInfo != null &&
+          serviceInfo.getNodeType() == HddsProtos.NodeType.OM) {
+        Map<String, Map<String, String>> omService = new HashMap<>();
+        omService.put(serviceInfo.getOmRoleInfo().getNodeId(),
+            new HashMap<String, String>() {{
+              put("serverRole", serviceInfo.getOmRoleInfo().getServerRole());
```

Comment:
```suggestion
              put("serverRole", omRoleInfo.getServerRole());
```

Reply:
I have incorporated your suggestions in the new method and the old one too. Thank you.